twilight-cache-inmemory/src/model/voice_state.rs

```diff
@@ -88,13 +90,11 @@ impl CachedVoiceState {
     pub const fn user_id(&self) -> Id<UserMarker> {
         self.user_id
     }
+}
 
-    /// Construct a cached voice state from its [`twilight_model`] form.
-    #[allow(clippy::missing_const_for_fn)]
-    pub(crate) fn from_model(
-        channel_id: Id<ChannelMarker>,
-        guild_id: Id<GuildMarker>,
-        voice_state: VoiceState,
+impl From<(Id<ChannelMarker>, Id<GuildMarker>, VoiceState)> for CachedVoiceState {
```

Comment:
we have these tupples in a few places. probably best to givve these explicit types?